react-tutorial/src/components/Caption.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Caption.css';
+
+const caption = () => {
```

Comment:
Well, logically, the component should be named 'Card' since the caption identity is more like a card-header. CSS classes should be renamed correspondingly.